extensions/wikia/Search/classes/Services/ESFandomSearchService.php

```diff
@@ -70,13 +72,16 @@ protected function consumeResponse( $response ) {
 		$results = [];
 
 		foreach ( $response as $item ) {
-			if ( isset( $item['_source'] ) ) {
+			if ( isset( $item['_source'] ) && isset( $item['_source']['title'] )
+			    && isset( $item['_source']['url'] ) ) {
 				$source = $item['_source'];
 
 				$results[] = [
 					'title' => html_entity_decode( $source['title'] ),
-					'excerpt' => isset( $source['excerpt'] ) ? html_entity_decode( $source['excerpt'] ) : '',
-					'vertical' => html_entity_decode( $source['vertical'] ),
+					'excerpt' => isset( $source['excerpt'] )
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "excerpt" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)